### PR TITLE
Paper: Add affiliations

### PR DESCRIPTION
**Pull request by EwoutH**
_Part of https://github.com/projectmesa/mesa/issues/2559._

Add affiliations to the JOSS paper.

image

- [Rendered paper (PDF download)](https://github.com/projectmesa/mesa/actions/runs/12529094250/artifacts/2368196930)

### Checklist

- [x] @EwoutH: Name, ORCID and affiliation correct
- [x] @quaquel: Name, ORCID and affiliation correct
- [ ] @Corvince:
   - [ ] Do you have an ORCID or do you want to make one?
   - [ ] Name, ORCID and affiliation correct
- [x] @tpike3: Name, ORCID and affiliation correct
- [x] @wang-boyu: Name, ORCID and affiliation correct
- [ ] @rht: Do you want to be included in the paper? If so, under what name / affiliation?
- [ ] @jackiekazil:
  - [ ] Do you want to go by Jackie or Jacqueline? 
  - [ ] Name, ORCID and affiliation correct

@projectmesa/maintainers please carefully check, and hit the checkbox if it's correct.

### CODE REVIEW

**Comment by jackiekazil** [comment]
I am struggling with the current paper structure without submitting something that is repetitive with previous papers, but also provides enough background for a reader to understand advancements.  I reached out to JOSS to try to get examples of papers that are version updates to see how others structured theirs -- https://github.com/openjournals/joss/issues/1403 -- specifically what level of context detail did they include?

I am interested to see what they highlight, because I can only see two paths right now -- 1) A lot of background info, which is repetitive with previous publications and will run long, but offers context for the advancements OR 2) A paper is almost a diff but in natural language -- where context maybe lacking.

**Comment by EwoutH** [comment]
@projectmesa/maintainers kind reminder to check your name/orcid/affiliation and hit the checkbox. If you are in doubt on something, need a little bit of time, don’t agree with something or don’t want to be on the paper that’s fine, but please just communicate something. Feels like talking to a wall sometimes.

File: paper/paper.md [hunk 1]
```diff
@@ -11,16 +11,36 @@ tags:
...
     orcid: 0009-0002-0805-3425
+    affiliation: "1"
   - name: Jan Kwakkel
     orcid: 0000-0001-9447-2954
+    affiliation: "1"
   - name: Vincent Hess
+    affiliation: "2"
   - name: Thomas D. Pike
     orcid: 0000-0003-1576-0283
+    affiliation: "3"
```
Comment (tpike3) [inline paper/paper.md:22]: Sorry I stink ---could you switch me to George Mason as well

Comment (EwoutH) [reply paper/paper.md:22]: No worries, done!

**Comment by jackiekazil** [comment]
Make mine the same as Tom's.

**Comment by EwoutH** [comment]
Merging for now, @Corvince and @rht, if you would like to be included in the final paper I would still need confirmation from you.

**Comment by Corvince** [comment]
Wow, sorry this whole paper somehow went completely unnoticed by me until your email. I read somewhere about a Joss paper, but want aware that it's about mesa itself and I could be one of the authors. Sorry! Is there a deadline? Hopefully I can contribute or at least review in the following days, but I am not sure about when exactly. I'll also look up my orcid

**Comment by EwoutH** [comment]
@Corvince No worries, thanks for getting back, and glad to hear from you! The bare minimum I need is confirmation you want to be on the paper, your name, affiliation and optionally ORCID is correct, and you are happy with the naming order. If you have time, feel free to review, all the information you need should be in the tracking issue (and otherwise please ask):

- https://github.com/projectmesa/mesa/issues/2559

The goal is to make our work on Mesa over the past years visible to the academic world, and one way to reach them (and rack up citations) is with a paper.

**Pull request merged**